bindings/Clutter/Clutter.overrides

```diff
@@ -0,0 +1,182 @@
+namespace Clutter
+
+delete-attr Cogl/Texture glib:get-type
+
+# Generated from clutter 1.26.4 with xsltproc Nullable.xslt Clutter-1.0.gir
```

Comment:
Unless you really need to keep support for old versions of the library with wrong nullable annotations I would recommend not including these overrides, as they make the overrides a bit harder to read.